tests/network/vmi_istio.go

```diff
@@ -183,6 +185,43 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 				}, tests.ContainerCompletionWaitTime, time.Second).Should(Succeed(), fmt.Sprintf("all containers should complete in source virt-launcher pod"))
 			})
 		})
+		Describe("SSH traffic", func() {
```

Comment:
Should be tested for vmi with and without ports.

Reply:
Done